images/public-log-vendor-matcher/README.md

```diff
@@ -0,0 +1,44 @@
+# Public log vendor matcher
+
+A Postgres-based k8s-infra data pipeline that produces BigQuery tables for reviewing Kubernetes Public artifact traffic in DataStudio.
+
+## Environment variables
+
+| Name                             | Default                                                | Description                                                                           |
+| -------------------------------- | ------------------------------------------------------ | ------------------------------------------------------------------------------------- |
+| `TZ`                             | ``                                                     | Container time zone                                                                   |
+| `GOOGLE_APPLICATION_CREDENTIALS` | ``                                                     | The path to the GCP service account json key                                          |
+| `GCP_PROJECT`                    | `k8s-infra-ii-sandbox`                                 | The project to target                                                                 |
```

Comment:
This feels ambiguous...
- the project that hosts `GCP_BIGQUERY_DATASET`?
- the project which is being billed for writing this data?